src/main/java/com/sapred/ordermanagerred/model/Currency.java

```diff
@@ -0,0 +1,11 @@
+package com.sapred.ordermanagerred.model;
+
+public enum Currency {
+    EURO(89.52907), SHEKEL(22.70258), FRANC(91.45229), RUBLE(0.97772), DOLLAR(81.98195);
```

Comment:
The enum value should be the short name of the currency not the exchange rate (the rate can be changed at any particular time so we should get it from an external web service)
EURO: EUR
SHEKEL: ILS
FRANC: CHF
RUBLE: RUB
DOLLAR: USD


https://www.eurochange.co.uk/travel/tips/world-currency-abbreviations-symbols-and-codes-travel-money